pkg/local_object_storage/pilorama/boltdb.go

```diff
@@ -883,10 +883,10 @@ func stateKey(key []byte, child Node) []byte {
 }
 
 func (t *boltForest) putState(b *bbolt.Bucket, key []byte, parent Node, timestamp Timestamp, meta []byte) error {
-	data := make([]byte, len(meta)+8+8)
-	binary.LittleEndian.PutUint64(data, parent)
-	binary.LittleEndian.PutUint64(data[8:], timestamp)
-	copy(data[16:], meta)
+	data := make([]byte, 0, len(meta)+8+8)
+	data = binary.LittleEndian.AppendUint64(data, parent)
+	data = binary.LittleEndian.AppendUint64(data, timestamp)
+	data = append(data, meta...)
```

Comment:
Not sure it's an improvement here, the old one is probably more performant.

Reply:
you mean exactly this line with `copy` before? (reverted `copy`)

Reply:
I mean whole chunk of course. It was pushing things into a known preallocated slice. Now it tries to `append()` at every step. It doesn't need to reallocate for obvious reasons, but it could've avoided these attempts.

Reply:
ok, reverted